src/runESLint.js

```diff
@@ -59,43 +59,32 @@ const pass = ({ start, end, testPath }) =>
     jestTestPath: testPath,
   });
 
-const runESLint = ({ testPath, config }, workerCallback) => {
-  try {
-    const start = +new Date();
+module.exports.runESLint = ({ testPath, config }) => {
+  const start = +new Date();
```

Comment:
`Date.now()`

Reply:
It was like that when I got here.  Should I still fix it?

Reply:
Ah, fair enough. Not required, but it’d be nice :-)